tool/gravity/cli/run.go

```diff
@@ -369,11 +369,13 @@ func Execute(g *Application, cmd string, extraArgs []string) error {
 	case g.StatusCmd.FullCommand():
 		printOptions := printOptions{
 			token:       *g.StatusCmd.Token,
-			tail:        *g.StatusCmd.Tail,
 			operationID: *g.StatusCmd.OperationID,
 			quiet:       *g.Silent,
 			format:      *g.StatusCmd.Output,
 		}
+		if *g.StatusCmd.Tail {
```

Comment:
I moved operation status tailing into a separate function as it needs to work with the actual cluster controller (not the 'local' operator from localenv/clusterenv) so it properly receives the operation log.

It also works around the issue with cluster state other than healthy when tailing operation logs:
```
[vagrant@node-1 ~]$ gravity status --operation-id=11c333ad-2bf4-412b-981f-c55d7e7f054f --tail
[ERROR]: Cluster is degraded
```